crates/proto/proto/responses.proto

```diff
@@ -17,7 +17,14 @@ message CheckNullifiersResponse {
 }
 
 message GetBlockHeaderByNumberResponse {
+    // The requested block header
     block_header.BlockHeader block_header = 1;
+
+    // Merkle path to verify the block's inclusion in the MMR at the returned `forest`
+    optional merkle.MerklePath proof = 2;
+
+    // Current value of the MMR forest
+    optional fixed32 forest = 3;
```

Comment:
Some nits about naming:

- I would probably rename `proof` into `mmr_path`.
- I would rename `forest` into something like `chain_length`.

Alternatively, we could introduce `MmrProof` message which would be similar to the `MmrProof` struct from `miden-crypto` - but `position` there would be somewhat redundant with the data in the block header itself. So, not sure if that's a good idea.

Reply:
I thought about reducing it to `MmrProof` but didn't because of what you mentioned. It does not seem necessary to me right now but I don't mind adding it if anyone has a different opinion.